pynxtools/dataconverter/convert.py

```diff
@@ -33,19 +33,14 @@
 from pynxtools.dataconverter.writer import Writer
 from pynxtools.dataconverter.template import Template
 from pynxtools.nexus import nexus
+from pynxtools.dataconverter.logger import logger as pynx_logger
 
 if sys.version_info >= (3, 10):
     from importlib.metadata import entry_points
 else:
     from importlib_metadata import entry_points
 
 
-logger = logging.getLogger(__name__)  # pylint: disable=C0103
-UNDOCUMENTED = 9
```

Comment:
> Why is undocumented removed? The idea was to have an additional level to just see the undocumented values. It can also be discussed to change this level to, e.g., have it above debug (level 10) and to align it with nomad.

I would say `undocumented` keys are the parts of the log info, need to be in to logger info level. Which already implemented in around line 224, making undocumented part of the log info.